python/ray/includes/global_state_accessor.pxd

```diff
@@ -158,7 +158,11 @@ cdef extern from * namespace "ray::gcs" nogil:
 
       auto context = cli->GetPrimaryContext();
       auto cmd = std::vector<std::string>{"DEL", key};
-      auto reply = context->RunArgvSync(cmd);
+      std::promise<std::shared_ptr<CallbackReply>> promise;
```

Comment:
worry a little bit about the deadlock if they are running in the same thread.

Reply:
oh, it's the util function. it's ok.

Reply:
Seems there is some test failure related to this, revert this part for now.